Wyam.Core/Modules/WriteFiles.cs

```diff
@@ -155,6 +155,7 @@ public virtual IEnumerable<IDocument> Execute(IReadOnlyList<IDocument> inputs, I
                             Metadata.Create(MetadataKeys.DestinationFileName, Path.GetFileName(path)),
                             Metadata.Create(MetadataKeys.DestinationFileDir, Path.GetDirectoryName(path)),
                             Metadata.Create(MetadataKeys.DestinationFilePath, path),
+                            Metadata.Create(MetadataKeys.RelativeDestinationFilePath, path.Substring(context.OutputFolder.Length)),
```

Comment:
I think it's ok if `WriteFiles` overwrites the metadata for `RelativeFilePath` etc. If there would be a usecase where the original pathes are needed *after* `WriteFiles`, they still can be copied before `WriteFiles` to metadata under a different name.